.obsidian/plugins/smart-connections/styles.css

```
@@ -0,0 +1,868 @@
+    .setting-item {
+      flex-direction: column;
+      align-items: flex-start;
+    }
+  }
+}
+
+.sc-typing-indicator {
+  display: flex;
+  align-items: center;
+  padding: 12px 16px;
+  display: none;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)